### PR TITLE
Add ontology to data model

### PR DESCRIPTION
**Pull request by brinxmat**
Adds the ontology to the model, we will need to address the issues with the ontology later, for now it is enough to have the basic mappings.

There is an obvious need for simplification here.

SEE NOTE BELOW: If you want to have fun, to see the work ahead, take a look at https://service.tib.eu/webvowl/#iri=https://raw.githubusercontent.com/BIBSYSDEV/nva-datamodel-java/add-ontology-to-data-model/nva-datamodel-java/src/main/resources/publication-ontology.ttl

NOTE: Use Filter > Degree of collapsing > 0

### CODE REVIEW

File: nva-datamodel-java/src/main/resources/publication-ontology.ttl [hunk 1]
```diff
@@ -0,0 +1,1744 @@
+@prefix nva: <https://nva.sikt.no/ontology/publication#> .
```
Comment (torarnet) [inline nva-datamodel-java/src/main/resources/publication-ontology.ttl:1]: Did you write all this manually?

File: nva-datamodel-java/src/test/java/no/unit/nva/OntologyTest.java [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    @Test
+    void shouldContainEveryVisibleClassOfModel() {
+        var ontologyClasses = extractClassesFromOntology();
+        var modelClasses = new ArrayList<>(getModelClasses()).toArray(String[]::new);
+        assertThat(ontologyClasses, hasItems(modelClasses));
+    }
+
+    @Test
+    void shouldContainEveryVisiblePropertyOfModel() {
+        var ontologyClasses = extractPropertiesFromOntology();
```
Comment (torarnet) [inline nva-datamodel-java/src/test/java/no/unit/nva/OntologyTest.java:76]: Names instead ontologyProperties and modelProperties? Not important though

**Review by torarnet — DISMISSED** [review]

**Review by ketilaa — DISMISSED** [review]

File: nva-datamodel-java/src/test/java/no/unit/nva/OntologyTest.java (hunk 2 above)
Comment (brinxmat) [reply nva-datamodel-java/src/test/java/no/unit/nva/OntologyTest.java]: Good catch

File: nva-datamodel-java/src/main/resources/publication-ontology.ttl (hunk 1 above)
Comment (brinxmat) [reply nva-datamodel-java/src/main/resources/publication-ontology.ttl:1]: Nope, I generated the majority of it — but had to do a lot of small fixes manually

Comment (brinxmat) [reply nva-datamodel-java/src/main/resources/publication-ontology.ttl:1]: In the future, I suspect we will have some annotations that aid generation.

**Review by torarnet — APPROVED** [review]

**Pull request merged**